src/handlers/mod.rs

```diff
@@ -569,5 +570,18 @@ impl OutputManagementHandler for State {
         self.niri.config.borrow_mut().outputs = conf;
         self.backend.on_output_config_changed(&mut self.niri);
     }
+
+    fn get_current_outputs(&self) -> OutputsConf {
+        let ipc_outputs = self.backend.ipc_outputs();
+        let ipc_outputs = ipc_outputs.lock().unwrap();
+        let mut res = HashMap::new();
+        for (name, output) in &self.niri.output_by_name {
+            let Some(conf) = ipc_outputs.get(name) else {
+                continue;
```

Comment:
I think this can have an error!() message too?

Reply:
yeah i had this change locally, didn't committed it, mb